content/community/automated-testing.md

```diff
@@ -0,0 +1,62 @@
+
+## About ChatOps
+
+We've setup some basic "ChatOps" style functionality to run tests. This is required in order to securely run infrastructure integration tests without exposing integration secrets.  More than this, it's a [hard limitation
+imposed by GitHub](https://www.google.com/search?q=github.community+run+on+forks+secrets) and the only way we can run the tests from Pull Requests originating from forks.
+
+Our tests run against some [testing infrastructure](https://github.com/cloudposse/testing.cloudposse.co) that we host in an isolated account on AWS, strictly for the purposes of testing infrastructure.
+
+Our ChatOps is powered by [GitHub Actions](https://github.com/features/actions) and the [slash-dispatch-command](https://github.com/peter-evans/slash-command-dispatch).
+
+The workflow is fined in the [`cloudposse/actions`](https://github.com/cloudposse/actions/blob/master/.github/workflows/test-command.yml) repository. The benefit with this is we have one place to control the testing
+workflow for all of our hundreds of terraform modules. The downside, however, with dispatched workflows is that the _workflows_ always run from the `master` branch. 
+
+
+## ChatOps Usage
+
+To run the tests, you'll need to issue some commands by posting GitHub comments on the PR. Note, only users that have been added to our [`cloudposse`](https://github.com/cloudposse) organization as a contributor are able to run these commands. If you're not a contributor,
+the commands will have no effect and there will be no error messages or warnings.
+
+
+| Command           | Description                                                      |
+| ----------------- | ---------------------------------------------------------------- |
+| `/test bats`      | Run bats automated tests                                         |
+| `/test readme`    | Verify the `README.md` is current and updated from `README.yaml` |
+| `/test terratest` | Run the `terratest` integration tests in `test/src`              |
+| `/test all`       | Run `readme`, `bats`, and `terratest` tests                      |
+
+
+All tests run from the [`cloudposse/actions`](https://github.com/cloudposse/actions/actions) repository.
+
+
+## ChatOps Configuration 
+
+If you're a contributor who wants to intialize one of our terraform modules, this is the process. Note, if a repo has already been configured for chatops, there's no need to proceed with these steps.
+
+To initialize one of our modules with chat, run the following commands:
+
+1. `git clone` the terraform module repository
+2. `cd $repo` to enter the repository directory
+3. `make init` to initialize the build-harness
+4. `make github/init` to write all the scaffolding
+5. `git add *` to add the changes
+6. `git rm -rf codefresh/` to remove legacy codefresh pipelines no longer needed
+7. Add the build badge to the `README.yaml` under the `badges` section. Also, remove any Codefresh badge, since that's also no longer needed. 
```

Comment:
```suggestion
7. Add the build badge to the `README.yaml` under the `badges` section. Also, remove any Codefresh badge, since that's also no longer needed
```